test/loadTest/README.md

```diff
@@ -53,11 +53,11 @@ Properties:
 		* THREADS        - number of concurrent threads (default: 1)
 		* RAMP_UP        - the amount of time for creating the total number of threads (default: 1)
 	    * TEST_TIME      - test duration time in seconds (default:30)
-		* SERVICE        - service header (default: update_ngsiv2)
+		* SERVICE        - service header (default: )
 		* SERVICE_PATH   - service path header (default: /)	    
 	    * ATTRIBUTES     - number of attributes per entity (default:1)
 	    * METADATA       - if true is appended a metadata in each attribute (default: false)
-	    * TIME_INSTANT   - if true is appended a "timeInstant" attribute in each entity (default: false)
+	    * TIME_INSTANT   - if true is appended a "timeInstant" attribute in each entity (default: true)
```

Comment:
Yes, the default value in the XML file is true:
https://github.com/telefonicaid/fiware-orion/blob/develop/test/loadTest/orion_updates_ngsiv2.jmx#L94